docs/cloud-native-security/index.rst

```diff
@@ -126,76 +122,48 @@ include: review of the customer’s runbook, summary of findings and
+
+Customers can choose between 2 billing models for Cloud Native Security – percentage of AWS infrastructure billing model or per-event billing model.
+
+Percentage of AWS infrastructure billing model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Customers that opt for this billing model will be charged a set percentage of their total AWS infrastructure costs on accounts that are in scope for Cloud Native Security. Charges will be based on usage of all AWS services and marketplace fees on accounts in scope for Cloud Native Security.
+
+In order to provide valuable service and to help with prioritization, a minimum severity level has been defined for each native AWS security product (source of findings). Rackspace will only ingest, and respond to, findings at or above the defined minimum severity level for each product. Findings with severities below this threshold will not be processed by Rackspace. The minimum severity levels for each product are defined as:
+
+* IAM Access Analyzer – **LOW**
+* Amazon GuardDuty – **MEDIUM**
+* AWS Security Hub (Security Standards) – **HIGH**
+
+Per-event billing model
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Customers that opt for this billing model will be charged a set fee for every security event Rackspace responds to. The monthly charge for the service is calculated as the product of the number of events multiplied by the cost per event, or the minimum monthly fee, whichever is greater.
+
+The per-event fee will apply to events (SIEM-correlated findings). For example, if the native security products generate five findings that are delivered to the SIEM and, after correlation, the SIEM generates two events (one for two related findings and one for three related findings), the customer will be charged for two events only.
+
+In order to give customers some control over costs (which have a direct correlation with the amount of findings Rackspace responds to) and to help with prioritization, we will define a minimum severity level for each customer. Rackspace will only respond to findings that are at or above the defined minimum severity level. Findings with lower severity level will not be processed by Rackspace and the customer won’t be charged for those. The minimum severity level will be agreed upon between the customer and Rackspace.
+
+Initial selection of the minimum severity level for Rackspace response can either be based on a review of the native security products in the customer’s environment (if they are already enabled) or simply be set to the highest severity level (i.e. Critical/High) to start with. As part of the regular monthly account reviews, the minimum severity level will be reviewed by Rackspace and the customer. When the current setting does not generate too many findings, we can reduce the minimum severity level and start responding to findings with a lower severity. Likewise, if the current setting is generating too many findings, we can increase the minimum severity level.
```

Comment:
... level (that is, `Critical` or `High`) to ...
... account reviews, Rackspace and the customer review the minimum severity level. ...

Reply:
updated